File: 1.FDP/2° Competencia/Java/Repeticion/Ejercicio4.java
import java.util.Scanner;

public class Ejercicio4 {
	public static void main (String[] args){
		Scanner scan= new Scanner(System.in);
		System.out.println("Calcular el area de un circulo.\n");
		double Radio, Area;
		int Respuesta;
		System.out.println("Cuantas veces deceas que se repita:");
		Respuesta= scan.nextInt();
		for (int i=0; i<Respuesta; i++) {
			System.out.println("Radio: ");
			Radio= scan.nextDouble();
			if (Radio > 0){
				Area= 3.14*(Radio*Radio);
				System.out.println("Area= " + Area);
				System.out.println("//////////////////////////////.");
			} else {
				System.out.println("Error en el radio");
				System.out.println("//////////////////////////");
			}	
		}
	}
}

File: 1.FDP/2° Competencia/C++/Repeticion/Ejercicio4.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main(){
	cout << "Calcular el area de un circulo.\n";
	double Radio, Area;
	cout << "Se repetira 3 veces.\n";
	for ( int i=0; i<3; i++) {
		cout << "Ingresa el radio:";
		cin >> Radio;
		if (Radio > 0){
			Area= 3.14 * (Radio * Radio);
			cout << "Area = " << Area << "\n";
			cout << "/////////////////////////////.\n";
		} else { 
			cout << "Error en el radio.\n";
			cout << "///////////////////////.\n";
		}
	}		
	return 0;
}